www/docs/en/dev/config_ref/index.md

```diff
@@ -39,222 +39,357 @@ Note that before version 3.3.1-0.2.0, the file existed at `app/www/config.xml`,
+   origin(string) | *Required* <br/> Defines the set of external domains the app is allowed to communicate with. 
+   The default value shown above allows it to access any server. 
+   See the Domain [Whitelist Guide](../guide/appdev/whitelist/index.html) for details. 
 
-## Global Preferences
+   Examples:
 
-The following global preferences apply to all platforms:
+    <widget ...>
+        <access origin="*"></content>
+    </widget>
+    
+    <widget ...>
+        <access origin="http://google.com"></content>
+    </widget>
+    
+    
+## allow-navigation
+   Controls which URLs the WebView itself can be navigated to. Applies to top-level navigations only.
+   
+   Attributes(type) | Description
+   ----------------- | ------------
+   href(string) | *Required* <br/> Defines the set of external domains the WebView is allowed to navigate to. 
+   See the cordova-plugin-whitelist [cordova-plugin-whitelist](https://github.com/apache/cordova-plugin-whitelist#navigation-whitelist) for details.
+   
+   Examples:
+   
+    <!-- Allow links to example.com -->
+    <allow-navigation href="http://example.com/*" />
+
+    <!-- Wildcards are allowed for the protocol, as a prefix to the host, or as a suffix to the path -->
+    <allow-navigation href="*://*.example.com/*" />
+   
+## allow-intent
+   Controls which URLs the app is allowed to ask the system to open. By default, no external URLs are allowed.
+   
+   Attributes(type) | Description
+   ----------------- | ------------
+   href(string) | *Required* <br/> Defines which URLs the app is allowed to ask the system to open. 
+   See the cordova-plugin-whitelist [cordova-plugin-whitelist](https://github.com/apache/cordova-plugin-whitelist#intent-whitelist) for details.  
+  
+   Examples:
+   
+    <allow-intent href="http://*/*" />
+    <allow-intent href="https://*/*" />
+    <allow-intent href="tel:*" />
+    <allow-intent href="sms:*" />
+  
+## engine
+   Specifies what platform to restore during a prepare. For more details, see the Platforms & Plugins Version Management.
+   
+   Attributes(type) | Description
+   ----------------- | ------------
+   name(string) | *Required* <br/> Name of the platform to be restored
+   spec(string) | *Required* <br/> Version of the platform to be restored
+   
+   Examples:
+
+    <engine name="android" spec="^4.0.0" />
+    
+## preference
+   Sets various optinos as pairs of name/value attributes. Each preference's name is case-insensitive. Many preferences are unique to specific platforms,
+   and will be indicated as such.  
+
+   Attributes(type) | Description
+   ----------------- | ------------
+   AllowInlineMediaPlayback(boolean) | *Default: false* <br/> **Platforms supported: iOS** <br/> Set to true to allow HTML5 media playback to appear inline within the screen layout, using browser-supplied controls rather than native controls. For this to work, add the webkit-playsinline attribute to any ```<video>``` elements. 
```

Comment:
We need to change the formatting for platform supported to ==<platform name>== . This makes it a part of 'mark' element which we apply custom css to on the website. I already updated the XML reference style guide to reflect this. Apologies for not conveying it earlier.